packages/fxa-settings/src/models/integrations/client-matching.ts

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const MONITOR_CLIENTIDS = [
```

Comment:
Can we query these from the database instead of hardcoding? I know they don't change often, but in the event one needs too change, or we need to add a new one, I thinks this replication of hard coding becomes an issue.

Reply:
I agree querying the database would be preferable but out of scope for the current ticket. I've filed https://mozilla-hub.atlassian.net/browse/FXA-8533